packages/core/src/physics/DynamicCollider.ts

```diff
@@ -46,6 +237,29 @@ export class DynamicCollider extends Collider {
     (<IDynamicCollider>this._nativeCollider).addTorque(torque);
   }
 
+  /**
+   * Moves kinematically controlled dynamic actors through the game world.
+   * @param position The desired position for the kinematic actor
+   * @param rotation The desired rotation for the kinematic actor
+   */
+  setKinematicTarget(position: Vector3, rotation: Quaternion) {
```

Comment:
It is more difficult to use for the case where only the target is set and the rotation is not considered.
Need for a fuller discussion.

Reply:
+1